### PR TITLE
Adds basic docs for install Tile

### PR DESCRIPTION
**Pull request by kul3r4**
#### WHAT
Add docs for Tile prompts

#### WHY
To promote this feature with developers

#### HOW
Add docs

#### Checklist :clipboard:
- [ ] Add explicit visibility modifier and explicit return types for public declarations
- [ ] Run spotless check
- [ ] Run tests
- [ ] Update metalava's signature text files

### CODE REVIEW

File: docs/datalayer-phone-ui.md [hunk 1]
```diff
@@ -110,6 +110,29 @@ following conditions:
...
+Please note that this feature is only supported on Pixel watches.
+
+The `shouldDisplayPrompt` method allows to check if the prompt should be displayed, based on the
+following conditions:
+- the Wear app is installed and it's tracking the status of installed Tiles. See
+[trackInstalledTiles](https://github.com/google/horologist/blob/main/datalayer/sample/wear/src/main/java/com/google/android/horologist/datalayer/sample/TileSync.kt#L35)
+in the sample app. The `trackInstalledTiles` has to be called when the app is launched, on 
+`TileService.onTileAddEvent` and `TileService.onTileRemoveEvent`. After the Wear app has been updated
+to support `trackInstalledTiles`, it has to be launched at least once before being able to track the 
+Tile installation status.
```
Comment (yschimke) [inline docs/datalayer-phone-ui.md:128]: Should we just trigger this ourselves? Avoid asking apps to do it?

File: docs/datalayer-phone-ui.md [hunk 2]
```diff
@@ -110,6 +110,29 @@ following conditions:
...
+- Wear OS companion app has version of 2.1.0.576785526 or above.
```
Comment (yschimke) [inline docs/datalayer-phone-ui.md:129]: Can we attach a date so people know that generally they should have this, and it's not future looking?

File: docs/datalayer-phone-ui.md [hunk 3]
```diff
@@ -110,6 +110,29 @@ following conditions:
...
+- We are using `getActiveTilesAsync` method which could omit some Tiles, 
+see more in the [doc](https://developer.android.com/reference/androidx/wear/tiles/TileService#getActiveTilesAsync(android.content.Context,java.util.concurrent.Executor)).
+
+Also notes that, if the users has more than one Pixel watches paired with the phone, the prompt
```
Comment (yschimke) [inline docs/datalayer-phone-ui.md:133]:
```suggestion
Also note that, if the user has more than one Pixel watches paired with the phone, the prompt
```

**Review by yschimke — APPROVED** [review]

File: docs/datalayer-phone-ui.md (hunk 3 above)
Comment (kul3r4) [reply docs/datalayer-phone-ui.md]: done

File: docs/datalayer-phone-ui.md (hunk 1 above)
Comment (kul3r4) [reply docs/datalayer-phone-ui.md:128]: Thanks, tracking in #2146.

File: docs/datalayer-phone-ui.md (hunk 2 above)
Comment (kul3r4) [reply docs/datalayer-phone-ui.md:129]: checking on this

Comment (kul3r4) [reply docs/datalayer-phone-ui.md]: added, PTAL

**Pull request merged**